src/sql/workbench/services/restore/common/constants.ts

```diff
@@ -10,3 +10,14 @@ export const fileFiltersSet: { label: string, filters: string[] }[] = [
 	{ label: localize('backup.filterBackupFiles', "Backup Files"), filters: ['*.bak', '*.trn', '*.log'] },
 	{ label: localize('backup.allFiles', "All Files"), filters: ['*'] }
 ];
+/**
+ * Backup media device type: https://docs.microsoft.com/en-us/dotnet/api/microsoft.sqlserver.management.smo.devicetype
+ */
+export enum DeviceType {
```

Comment:
Could you actually move this into https://github.com/microsoft/azuredatastudio/blob/506c6a5e5f8423d3249670f55b1466b1980a5f03/src/sql/workbench/contrib/backup/common/constants.ts and then remove the constant values here? https://github.com/microsoft/azuredatastudio/blob/506c6a5e5f8423d3249670f55b1466b1980a5f03/src/sql/workbench/contrib/backup/common/constants.ts#L16

(and if you're feeling generous making an enum for the physical device type would be awesome too...)

Reply:
I had to add new rules to the eslint file because restore files couldn't access [azuredatastudio/src/sql/workbench/contrib/backup/common/constants.ts](https://github.com/microsoft/azuredatastudio/blob/506c6a5e5f8423d3249670f55b1466b1980a5f03/src/sql/workbench/contrib/backup/common/constants.ts#L16)